File: frontend/formfilling/libs/output.js
const colors = require('colors');


// print_out is a interface for future GUI as a replacement of console.log
function print(message, style) {
    switch (style) {
        case 'info':
            console.log(`${colors.blue(message)}`);
            break;
        case 'error':
            console.log(`${colors.red(message)}`);
            break;
        case 'success':
            console.log(`${colors.green(message)}`);
            break;
        case 'warning':
            console.log(`${colors.yellow(message)}`);
            break;
        default:
            const darkWhite = colors.white.dim;
            console.log(`${darkWhite(message)}`);
            break;

    }
}


module.exports = {
    print
};

File: client/client/app/util/output.go
package util

import (
	"os"

	"github.com/fatih/color"
	"github.com/olekukonko/tablewriter"
)

func PrintTable(data [][]string) {
	table := tablewriter.NewWriter(os.Stdout)
	table.SetHeader(data[0])
	data = data[1:]
	for _, row := range data {
		table.Append(row)
	}
	table.Render()
}

func ConsolePrint(message, format string) {
	switch format {
	case "info":
		color.Blue(message)
	case "error":
		color.Red(message)
	case "warning":
		color.Yellow(message)
	case "success":
		color.Green(message)
	default:
		color.White(message)
	}
}
